src/utils/transform/get-esbuild-options.ts

```diff
@@ -10,18 +10,25 @@ export const baseConfig = Object.freeze({
 	loader: 'default',
 });
 
+const isNodeDebuggerEnabled = process.execArgv.some((flag) => {
+
+	// match Node.js debugger flags
+	// https://nodejs.org/api/cli.html#--inspecthostport
+	return /--inspect(-brk|-port|-publish-uid|-wait)?/.test(flag)
```

Comment:
Can you hardcode the combinations instead?

Regex is slow and hard coding it will be more readable/searchable

Reply:
Should I go like this

```js
flag.startsWith('--inspect-brk') || flag.startsWith('--inspect-port') || flag.startsWith('--inspect-publish-uid') || flag.startsWith('--inspect-wait') || flag.startsWith('--inspect')
```

or just this

```js
flag.startsWith('--inspect')
```

Because these are both valid parameters

```sh
tsx --inspect test.ts
tsx --inspect=9229 test.ts
tsx --inspect=127.0.0.1:9229 test.ts

tsx --inspect-brk test.ts
tsx --inspect-brk=9229 test.ts
tsx --inspect-brk=127.0.0.1:9229 test.ts
```

Reply:
Ah I see, I guess it's more complicated than I expected. Thanks for illustrating the problem.

Let's stick to a regex but can you move it out of the loop so it doesn't get recreated every time, and do something like this instead:
`^--inspect(-brk|-port|-publish-uid|-wait)(=|$)`

Reply:
I end up with `/^--inspect(?:-brk|-port|-publish-uid|-wait)?(?:=|$)/`

Because the capturing group is not used, `(?:...)` is used to solve lint errors like `Capturing group number 1 is defined but never used`